basis/io/directories/directories.factor

```diff
@@ -68,20 +68,12 @@ HOOK: delete-directory io-backend ( path -- )
 : to-directory ( from to -- from to' )
     over file-name append-path ;
 
-! Moving and renaming files
-HOOK: move-file io-backend ( from to -- )
-
-: move-file-into ( from to -- )
-    to-directory move-file ;
-
-: move-files-into ( files to -- )
-    '[ _ move-file-into ] each ;
-
 ! Copying files
 HOOK: copy-file io-backend ( from to -- )
 
 M: object copy-file
-    make-parent-directories binary <file-writer> [
+    [ normalize-path ] bi@ make-parent-directories
```

Comment:
it's not necessary here because you are using ``make-parent-directories`` which calls ``make-directory`` which normalizes, and you are using the ``<file-reader>`` and ``<file-writer>`` which should sanitize if necessary any user paths.  so we should remove these calls to ``normalize-path``.